tools/openshift-ci/verification_tests_ci_entrypoint.sh

```diff
@@ -0,0 +1,202 @@
+  features/web/deployments.feature:100 \
+  features/cli/oc_help.feature:5 \
+  features/cli/build.feature:184 \
+  features/cli/oc_import_image.feature:71 \
+  features/build/buildlogic.feature:151 \
+  features/cli/build.feature:5 \
+  features/cli/oc_idle.feature:215 \
+  features/cli/projects.feature:137 \
+  features/cli/deploy.feature:445 \
+  features/cli/oc_set_probe.feature:5 \
+  features/cli/policy.feature:496 \
+  features/cli/oc_set_probe.feature:107 \
+  features/cli/configmap.feature:295 \
+  features/build/buildlogic.feature:53 \
+  features/cli/downwardapi.feature:18 \
+  features/web/deployments.feature:6 \
+  features/cli/scale.feature:5 \
+  features/cli/pod.feature:143 \
+  features/cli/oc_set_probe.feature:62 \
+  features/cli/deploy.feature:370 \
+  features/cli/configmap.feature:272 \
+  features/cli/route.feature:5 \
+  features/cli/oc_set_env.feature:58 \
+  features/cli/deploy.feature:763 \
+  features/build/stibuild.feature:59 \
+  features/web/service-catalog/project.feature:5 \
+  features/online/metrics.feature:5 \
+  features/admin/scheduler.feature:5 \
+  features/cli/oc_secrets.feature:5 \
+  features/web/check_page.feature:26 \
+  features/cli/deploy.feature:272 \
+  features/cli/deploy.feature:5 \
+  features/web/pod.feature:5 \
+  features/rest/webhook.feature:52 \
+  features/cli/configmap.feature:37 \
+  features/cli/secrets.feature:413 \
+  features/cli/build.feature:361 \
+  features/cli/oc_set_env.feature:5 \
+  features/build/dockerbuild.feature:5 \
+  features/cli/oc_set_build_hook.feature:6 \
+  features/web/scale.feature:5 \
+  features/cli/configmap.feature:75 \
+  features/cli/oc_tag.feature:5 \
+  features/cli/deploy.feature:188 \
+  features/cli/secrets.feature:62 \
+  features/cli/oc_secrets.feature:168 \
+  features/cli/deployment.feature:374 \
+  features/cli/deploy.feature:662 \
+  features/web/overview.feature:181 \
+  features/build/env.feature:5 \
+  features/web/check_page.feature:42 \
+  features/cli/deployment.feature:5 \
+  features/cli/deployment.feature:78 \
+  features/cli/oc_idle.feature:5 \
+  features/cli/pod.feature:161 \
+  features/cli/job.feature:84 \
+  features/build/buildlogic.feature:17 \
+  features/cli/build.feature:71 \
+  features/web/routes.feature:37 \
+  features/cli/serviceaccount.feature:30 \
+  features/web/create.feature:256 \
+  features/build/dockerbuild.feature:111 \
+  features/cli/resources.feature:5 \
+  features/web/configmap.feature:5 \
+  features/cli/deploy.feature:150 \
+  features/cli/policy.feature:379 \
+  features/cli/deploy.feature:321 \
+  features/cli/configmap.feature:115 \
+  features/cli/configmap.feature:179 \
+  features/cli/event.feature:92 \
+  features/containers/containers.feature:7 \
+  features/cli/job.feature:192 \
+  features/networking/pod.feature:42 \
+  features/cli/policy.feature:408 \
+  features/images/jenkins.feature:382 \
+  features/web/configmap.feature:63 \
+  features/cli/oc_set_deployment_hook.feature:1
```

Comment:
we need to generate this list on the fly based on case_id tag. Otherwise changes to the files can easily change lines of scenarios thus messing up the run completely.

Reply:
agreed; spoke with @kwoodson about this very issue yesterday. We may want to add this functionality as a post-merge action, to automatically add testcases here once merged (if they're dedicated related tests)

Reply:
@cjryan what do you mena "post-merge action"? What I request is to call a script before running the tests that will generate this command line instead of having these file:num hardcoded. The script would be based on hardcoded list of test case IDs, that we (QE) will update regularly.